introduction.md

```diff
@@ -1,2 +1,6 @@
 Hello, my name is Will and I am excited to learn more about Git. My
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?